### DISCUSSION TITLE
Github requiring E-Mail-Domain which is unavailible to students

### DISCUSSION THREAD

**Discussion by GuenniGuenther** [community/community#68295] [GitHub Education] [answered]
### Hi everyone,

Github is requiring a specific E-Mail-Domain which is not availible to students, we use a different Domain (Teachers use the websites domain, students is ***.onmicrosoft.com
I've already contacted Github Support 2 months ago, but never got a response besides the default bot response.
Does anyone have suggestions about how to proceed? The new school year has started a few weeks ago and it is quite annoying not to be able to use Github and the Benefits. 
![grafik](https://github.com/community/community/assets/121099682/19055794-dc5e-4fe5-a790-864e786ce5a4)

**Comment by milindaShehan** [comment]
- Try contacting GitHub support again. You can do this by submitting a ticket on the GitHub website. Be sure to include as much information as possible in your ticket, such as the specific email domain that your school uses and the benefits that you are missing out on.
- Try contacting your school's IT department. They may be able to help you get your school approved to use GitHub with your school's email domain.
- Try using a different email address to create a GitHub account. You can use a personal email address or a free email address from a service like Gmail or Hotmail.

**Reply by GuenniGuenther** [reply]
I guess we will contact support again. 
Our IT department does not want to have anything to do with external resources and, as seen in the screenshot, we _can't_ use a different e-Mail address.

**Comment by GuenniGuenther** [comment, marked as answer]
We've reached out to Support again, they've resolved the issue.
